plan/plan.go

```diff
@@ -186,13 +189,40 @@ func (p *Plan) GetInstance(path string, line, char int) (*loader.Instance, error
 	} else {
 		i, found := p.imports[def.Pkg()]
 		if !found {
-			return nil, fmt.Errorf("imported package %s not registered in plan", def.Def())
+			if i, err = p.findInstanceAlias(path, def.Pkg()); err != nil {
+				return nil, fmt.Errorf("imported package %s not registered in plan", def.Def())
+			}
 		}
 
 		return i, nil
 	}
 }
 
+// findInstanceAlias returns the instance of a package alias
+func (p *Plan) findInstanceAlias(path string, pkgAlias string) (*loader.Instance, error) {
+	p.log.Debugf("Looking for pkg alias: %s", pkgAlias)
+
+	p.muFiles.RLock()
+	defer p.muFiles.RUnlock()
+
+	f, ok := p.files[path]
+	if !ok {
+		return nil, fmt.Errorf("file not registered")
+	}
+
+	if importPath, ok := f.AliasImportPath(pkgAlias); ok {
+		for pkgName, i := range p.imports {
+			if i.ImportPath == importPath {
+				if i, found := p.imports[pkgName]; found {
+					return i, nil
+				}
```

Comment:
`i` is already set to the current instance no? Are Those 3 lines useful?

Reply:
Absolutely not :) 

Updated

Reply:
Nice, thanks you :rocket: